upload/system/library/paypro/language/dutch.php

```diff
@@ -70,7 +66,7 @@
 
 $_['error_permission'] = 'Toegangsfout';
 $_['error_api_key'] = 'Geldige API key is verplicht';
-$_['info_product_id'] = 'Required when using MasterCard, Visa or Sofort payment methods';
+$_['info_product_id'] = 'Required when using MasterCard, Visa or Sofort';
```

Comment:
Why we update the text here?

Reply:
i thought `sofort payment methods` referred to `sofort/digital` and `sofort/physical` i could have seen it wrong at the time. i thought `payment methods` was together with `sofort` and since `sofort` is now `one` method it doesnt make sense to put it there :) if i am wrong let me know and will be updated!

Reply:
Actually it is not necessary anymore to have a `product_id` for Mastercard, Visa or Sofort. So this line can be removed completely from the plugin.